lib/Partitioner/Partitioner.cpp

```diff
@@ -546,7 +546,7 @@ void Partitioner::doPartitioning(Function *F, NodeToFunctionMap &mapping) {
   }
 }
 
-DAGListTy &Partitioner::Partition() {
+llvm::Error Partitioner::Partition() {
```

Comment:
Do we usually use `llvm::Error` or `llvm::Expected` in such cases, where a function is supposed to return a value?

Reply:
We use llmv::Error. If there is no error, the function will return `llvm::Error::success();`